packages/cli/generators/import-lb3-models/index.js

```diff
@@ -136,6 +148,20 @@ Learn more at https://loopback.io/doc/en/lb4/Importing-LB3-models.html
     );
     debug('LB4 model data', templateData);
 
+    if (!templateData.isModelBaseBuiltin) {
+      const baseName = templateData.modelBaseClass;
+      if (
+        !this.existingLb4ModelNames.includes(baseName) &&
+        !this.modelNames.includes(baseName)
+      ) {
+        this.log(
+          'Adding %s (base of %s) to the list of imported models.',
+          chalk.yellow(baseName),
+          chalk.yellow(name),
+        );
```

Comment:
Here is an example console output.

```
$ lb4 import-lb3-models ~/src/loopback/next/packages/cli/test/fixtures/import-lb3-models/app-using-model-inheritance.js

WARNING: This command is experimental and not feature-complete yet.
Learn more at https://loopback.io/doc/en/lb4/Importing-LB3-models.html

? Select models to import: Customer
Model Customer will be created in src/models/customer.model.ts

Adding UserBase (base of Customer) to the list of imported models.
Model UserBase will be created in src/models/user-base.model.ts

Adding User (base of UserBase) to the list of imported models.
Model User will be created in src/models/user.model.ts

Import of model relations is not supported yet. Skipping the following relations: accessTokens
Ignoring the following unsupported settings: acls
   create src/models/customer.model.ts
   create src/models/user-base.model.ts
   create src/models/user.model.ts
   update src/models/index.ts
   update src/models/index.ts
   update src/models/index.ts
```

Reply:
So just making sure I understand correctly, we're importing `User` because `UserBase` inherits from it? And do we always eventually want one model to extend from a built in model (e.g. in this case `User` inheriting from `Entity`)?

Reply:
> We're importing `User` because `UserBase` inherits from it? 

Yes, exactly :+1:

> And do we always eventually want one model to extend from a built in model (e.g. in this case `User` inheriting from `Entity`)?

Yes. Every LoopBack 3 model has the LB3 built-in `Model` at the top of the inheritance chain. Some models are inheriting from `PersistedModel` (which inherits from `Model` and is mapped to `Entity` in LB4), some models are inheriting from `KeyValueModel` (which inherits from `Model` and is probably mapped to `KeyValueModel` in LB4, I am not sure).